src/test/pile.t.sol

```diff
@@ -20,13 +20,20 @@ import "ds-test/test.sol";
 import "../pile.sol";
 import "./mock/token.sol";
 
+contract Hevm {
+    function warp(uint256) public;
+}
+
 
 contract PileTest is DSTest {
     Pile pile;
     TokenMock tkn;
 
-    function setUp() public {
+    Hevm hevm;
 
+    function setUp() public {
+        hevm = Hevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+        hevm.warp(0);
```

Comment:
Can you test with something other than warp(0)? Check this out: https://github.com/makerdao/dss/pull/45/files#diff-ba2ea047fd34da8c18c72fcc967b6e4bL115 (setting it to 0 was the problem why we didn't find the rho issue)